client/src/modules/reports/generate/stock_consumption_graph_report/stock_consumption_graph_report.html

```diff
@@ -0,0 +1,64 @@
+<bh-report-preview
+  ng-if="ReportConfigCtrl.previewGenerated"
+  source-document="ReportConfigCtrl.previewResult"
+  on-clear-callback="ReportConfigCtrl.clearPreview()"
+  on-save-callback="ReportConfigCtrl.requestSaveAs()">
+</bh-report-preview>
+
+<div ng-show="!ReportConfigCtrl.previewGenerated">
+  <div class="row">
+    <div class="col-md-12">
+      <h3 translate>REPORT.STOCK_CONSUMPTION_GRAPH_REPORT.TITLE</h3>
+      <p class="text-info" translate>REPORT.STOCK_CONSUMPTION_GRAPH_REPORT.DESCRIPTION</p>
+    </div>
+  </div>
+
+  <div class="row" style="margin-top : 10px">
+    <div class="col-md-6">
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <span translate>REPORT.UTIL.OPTIONS</span>
+        </div>
+
+        <div class="panel-body">
+
+        <form name="ConfigForm" bh-submit="ReportConfigCtrl.preview(ConfigForm)" novalidate autocomplete="off">
+
+          
+          <!-- select depot -->
+          <bh-depot-select
+            depot-uuid="ReportConfigCtrl.depot.uuid"
+            on-select-callback="ReportConfigCtrl.onSelectDepot(depot)">
+            <bh-clear on-clear="ReportConfigCtrl.clear('depot_uuid')"></bh-clear>
+          </bh-depot-select>
+
+          <!-- select inventory -->
+          <bh-inventory-select
+            inventory-uuid="ReportConfigCtrl.inventory.uuid"
+            on-select-callback="ReportConfigCtrl.onSelectInventory(inventory)"
+            only-consumable="true">
+            <bh-clear on-clear="ReportConfigCtrl.clear('inventory_uuid')"></bh-clear>
+          </bh-inventory-select>
+
+          
+          <bh-fiscal-year-select
+            fiscal-id="ReportConfigCtrl.reportDetails.fiscal_id"
+            on-select-fiscal-callback="ReportConfigCtrl.onSelectFiscalYear(fiscalYear)">
+          </bh-fiscal-year-select>
+
+          <bh-period-selection
```

Comment:
Just FYI - we have the [`bhFiscalYearPeriodSelect`](https://github.com/IMA-WorldHealth/bhima/blob/master/client/src/js/components/bhFiscalYearPeriodSelect/bhFiscalYearPeriodSelect.js) that groups these two (fiscal year and period) selection. It might make your logic easier in the future to just use that.

Reply:
As the depot selection is optional we can keep ```Stock Consumption's graph report``` as title

Reply:
I think this response was to another comment.  That makes sense.  I guess I didn't realize the depot was optional, but that makes sense from the code now.